platform_versioned_docs/version-24.1.0/enterprise/docker-compose.mdx

```diff
@@ -40,3 +40,13 @@ The DB or Redis volume is persistent after a Docker restart by default. Use the
 :::note
 For more information on configuration, see [Configuration options](./configuration/overview.mdx).
 :::
+
+## Optional features
+
+### Data Studios
+
+Seqera Platform provides an interactive tertiary analysis environment that you can enable for organizational workspaces. For more information enabling Data Studios, see [Data Studios deployment](./enterprise/data-studios.mdx).
+
+:::note
+Data Studios is currently in **public preview** and is available from Seqera Platform v24.1. If you experience any problems during the deployment process please contact your account executive. Data Studios in Enterprise is not enabled by default. You can enable Data Studios in the [environment variables configuration](./enterprise/configuration/overview#opt-in-seqera-features).
```

Comment:
Account executive or support?

Reply:
Also, enabling Data Studios involves more than just the env var, as it requires the deployment steps referenced above. So this admonition may be misleading.